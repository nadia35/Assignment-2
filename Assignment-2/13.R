repeatYourself <- function(vec, num_times){

  if (num_times > 0){
  for (i in 1:num_times){
    print(vec[1])
  }
}
}

