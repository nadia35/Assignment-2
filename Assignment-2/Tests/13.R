source("13.R")

repeatYourself("Hello, world!", 3)
repeatYourself("I like repeating myself", 10)
repeatYourself("But not this phrase", 0)
repeatYourself("Goodbye, world!", 1)