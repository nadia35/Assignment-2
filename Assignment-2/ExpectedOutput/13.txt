[1] "Hello, world!"
[1] "Hello, world!"
[1] "Hello, world!"
[1] "I like repeating myself"
[1] "I like repeating myself"
[1] "I like repeating myself"
[1] "I like repeating myself"
[1] "I like repeating myself"
[1] "I like repeating myself"
[1] "I like repeating myself"
[1] "I like repeating myself"
[1] "I like repeating myself"
[1] "I like repeating myself"
[1] "Goodbye, world!"
